modules/nlu/src/backend/index.ts

```diff
@@ -34,19 +45,39 @@ export const initializeLangServer = async (bp: typeof sdk) => {
 
   const { validProvidersCount, validLanguages } = langProvider.getHealth()
 
+  // check if we can remove this
```

Comment:
That is used to display a warning on the studio if no language is correctly set or if there's an error with the provider, so the user is aware of any issue while talking with the bot. I think it should stay there